CONTRIBUTING.md

```diff
@@ -338,6 +348,45 @@ This involves the following steps:
    ```bash
    >>> git push origin --delete feature-add-new-important-headers
    ```
+
+## Code review
+Generally, pull requests should be of a "digestible" size so that any issues arising during a code review can be resolved within a few iterations. Interact with the code reviewer by whichever means is easiest for you, but use the "comments" feature on the Github webpages in the first instance so there's a record of the issues raised. Sometimes it'll be easier to follow up via zoom or face-to-face discussions but in any case, the discussion should ultimately be marked as resolved, and the changes made should be transparent. "All done" is fine; more detail is required if suggestions were (jointly) deemed to be inappropriate. No need to over-engineer any of this -- small pull requests can just be merged in. However, the workflow described below should be sufficiently robust to deal even with "monster" pull requests.
+
+The general workflow for approving a pull request is as follows:
+
+1. **Reviewer:** Go to the "Files changed" tab for the pull request
+
+      ![](doc/README/files_changed_tab.png)
+
+   and inspect the changes made.
+
+ 1. **Reviewer:** Add comments/requests for change by leaving a comment near the relevant part of the code. Hover over the line number until the blue plus sign appears
+    ![](doc/README/plus_sign.png)
```

Comment:
This picture and several after it are appearing inline and [it looks a bit odd](https://github.com/MatthiasHeilManchester/oomph-lib/blob/update_workflow_instructions/CONTRIBUTING.md#code-review). If you add a newline to separate the text and image, it should make it display not inline.

Reply:
Done